microsoft-edge/web-platform/sidebar.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Develop for the sidebar
+description: Learn to develop experiences for the sidebar in Microsoft Edge, by either installing a Progressive Web App, or using the extension sidebar API.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 07/24/2023
+---
+# Develop for the sidebar
+
+The sidebar in Microsoft Edge is a persistent pane located on the side of the browser, which coexists with the primary content of the browser. The sidebar allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs. With the sidebar in Microsoft Edge, users can access the productivity tools they need while staying in their workflow.
+
+![The sidebar in Microsoft Edge, showing useful tools, alongside the main tab](./sidebar-images/sidebar.png)
+
+As a developer, you can leverage the sidebar for your own experiences in two ways:
+
+* By adapting your existing Progressive Web App (PWA) to run in the sidebar.
+* By making use of the Sidebar API in your Microsoft Edge extension.
+
+
+<!-- ====================================================================== -->
+## Choose the right approach
+
+The two different ways in which you can leverage the sidebar are designed to meet different developer needs. The following are the key differences between sidebar apps and sidebar extensions:
+
+|Approach | Sidebar app | Sidebar extension |
```

Comment:
```suggestion
| Aspect | Sidebar app | Sidebar extension |
```
instead of pointing to the right, point down.  What's below in this column isn't 7 approaches, but 7 aspects of the two approaches.